actions/actions.py

```diff
@@ -7,21 +7,43 @@
 
 # This is a simple example for a custom action which utters "Hello World!"
 
-# from typing import Any, Text, Dict, List
-#
-# from rasa_sdk import Action, Tracker
-# from rasa_sdk.executor import CollectingDispatcher
-#
-#
-# class ActionHelloWorld(Action):
-#
-#     def name(self) -> Text:
-#         return "action_hello_world"
-#
-#     def run(self, dispatcher: CollectingDispatcher,
-#             tracker: Tracker,
-#             domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
-#
-#         dispatcher.utter_message(text="Hello World!")
-#
-#         return []
+from typing import Any, Text, Dict, List
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.executor import CollectingDispatcher
+import tensorflow as tf
+from transformers import TFAutoModelForQuestionAnswering
+from transformers import AutoTokenizer
+import src.config as cf
+from src.preprocessing import load_context_for_inference
+
+
+class ActionCovid(Action):
+
+    def name(self) -> Text:
+        return "action_covid"
```

Comment:
this can benefit from a more descriptive name such as `action_answer_covid_question`